spinta/ufuncs/loadbuilder/ufuncs.py

```diff
@@ -70,3 +70,22 @@ def input_(env: LoadBuilder, expr: Expr) -> None:
     prep_value = args[0] if args else NA
 
     env.param.soap_body = {env.this: prep_value}
+
+
+@ufunc.resolver(LoadBuilder, Expr)
+def header(env: LoadBuilder, expr: Expr) -> None:
+    args, kwargs = expr.resolve(env)
+    if len(args) != 1:
+        raise InvalidArgumentInExpression(arguments=args, expr="header")
+
+    return env.call("header", env.this, args[0])
```

Comment:
Confusing `return` (if return typehint is correct)
```suggestion
    env.call("header", env.this, args[0])
```

Reply:
This call always returns None. Adding `return` makes it more explicit.